elixir-mode.el

```diff
@@ -265,7 +265,7 @@ is used to limit the scan."
   (unless (elixir-syntax-in-string-or-comment-p)
     (let ((heredoc-p (save-excursion
                        (goto-char (match-beginning 0))
-                       (looking-at-p "~[sS]\\(?:'''\\|\"\"\"\\)"))))
+                       (looking-at-p "~[BCDELNRSTUbcersw]\\(?:'''\\|\"\"\"\\)"))))
```

Comment:
@axelson we should generate tests for these new sigils as it's really easy to break and not notice day-to-day.

Reply:
Yes, that's a good point and it would be good to include tests for them